ArduSub/defines.h

```diff
@@ -39,7 +39,8 @@ enum control_mode_t {
     CIRCLE =        7,  // automatic circular flight with automatic throttle
     SURFACE =       9,  // automatically return to surface, pilot maintains horizontal control
     POSHOLD =      16,  // automatic position hold with manual override, with automatic throttle
-    MANUAL =       19   // Pass-through input with no stabilization
+    MANUAL =       19,   // Pass-through input with no stabilization
+    RAW =          20   // Pass-through rc pwms to motors
```

Comment:
Here is the id of the new mode, it might be changed to a bigger number in the future.

Reply:
We might change it now, because it is already occupied with motor test mode if I'm not wrong.